### ISSUE TITLE
Add accessibility testing

### ISSUE THREAD

**Issue by anjackson** [ukwa/docker-robot-framework#5]
While accessibility testing cannot be fully automated, there are automation tools that can spot the subset of problems that are automatable, and this is a useful regression test while we update or refactor the site.

As a proof of concept, I have integrated the [Pa11y](https://pa11y.org/) tool via a custom Robot Framework keyword library ([`Pa11yLibrary.py`](https://github.com/ukwa/docker-robot-framework/blob/main/Pa11yLibrary.py)). This allows test to be run on selected pages, and integrates the results into the overall Robot Framework service tests. It runs two a11y testing tools, `axe` and `htmlcs`, and the tests fail if either find anything they consider errors. Warnings are also included but do not make the tests fail.

https://github.com/ukwa/docker-robot-framework/blob/8f3bf35a56981472cdf11383eeb422c11d502216/tests/website/a11y.robot#L1-L11

Right now, the tests do not pass. In particular, the footer markup appears to need some tidying up. As this is early-days, I'm not sure if the results are accurate, so I've configured the test suite to mark `a11y` tagged tests as non-critical, so the failures are skipped.

To Do:

- [x] Check with @min2ha if this tool and approach is acceptable.
- [ ] Add the other pages to the testing, using one or two representative pages for each dynamic page (e.g. search, archival playback).
- [ ] For the homepapge at least, also switch to high-contrast mode and run Pa11y there too.

**Comment by anjackson** [comment]
Awaiting on porting of tests #3 so these tests can be run cleanly from Airflow.

**Comment by anjackson** [comment]
With b5847c90985b8c29cece007a2ca26d7601b6b696 the various pages are covered.  The flicking-to-high-contrast mode is a bit more complicated, so I'm going to separate that out for now.
